src/cpu/flexcpu/inflight_inst.hh

```diff
@@ -109,6 +109,18 @@ class InflightInst : public ExecContext,
         Tick commitTick;
         // If squashed, when was this InflightInst squashed?
         Tick squashTick;
+
+        // Constructor for the struct. This sets all Tick's to MaxTick, which
+        // means if a Tick hasn't been recorded, its value will be MaxTick.
+        TimingRecord(): creationTick(MaxTick),
```

Comment:
Sorry for the minor comment, but I think we should start setting these defaults in the definitions of the member variables.

For instance:

```
Tick squashTick = MaxTick;
```

This will save lots of lines of code.

Reply:
Will change that. But AFAIK, this way of setting the default value for a struct are supported after C++ 11 according to [this link](https://stackoverflow.com/questions/28519061/how-to-define-a-struct-with-default-values). So anybody having an ancient C++ compiler that doesn't support C++ 11 standard or later won't get gem5 compiled ...